dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -874,8 +874,7 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     sign_in @student
 
     get :show, script_id: sl.script, stage_id: sl.stage, id: sl, solution: true
-
-    assert_response :forbidden
+    assert_select '.teacher-panel', 0
```

Comment:
So the URL would be https://studio.code.org/s/20-hour/stage/2/puzzle/11?solution=true but I would see the same content as https://studio.code.org/s/20-hour/stage/2/puzzle/11?